tests/testthat/helper.R

```diff
@@ -0,0 +1,85 @@
+if (!requireNamespace("RBackend", quietly = TRUE)) {
+  stop("RBackend is needed to run the tests. Please install it.", call. = FALSE)
+}
+
+if (!requireNamespace("rjson", quietly = TRUE)) {
+  stop("rjson is needed to run the tests. Please install it.", call. = FALSE)
+}
+
+test_it <- function(lst) {
```

Comment:
rather than running the backend in this function, it seems like we could create a State object, so the backend won't slow down development. If tests using the backend seem necessary, they could be marked (future music).

see the master branch's [setup_state function](https://github.com/datacamp/testwhat.ext/blob/master/tests/testthat/helper-setup_state.R), and [example unit tests](https://github.com/datacamp/testwhat.ext/blob/master/tests/testthat/test_examples.R#L13-L18).

Running tests in testwhat has been very cumbersome, since using the backend makes it hard to reason about why a test failed. `sqlwhat` and `shellwhat` don't use the backends, except for a couple sanity checks that aren't run on travis, and it has been much easier.

What do you think?

Reply:
Fixed for roxygen tests via https://github.com/datacamp/testwhat.ext/commit/261cb4e8a5ca47acbf9049889ce8848010423bb4.

Reply:
Fixed for DESCRIPTION tests via https://github.com/datacamp/testwhat.ext/commit/0dd5c845eee374333a6e2208792d259cd0c84911.